yt/data_objects/particle_trajectories.py

```diff
@@ -335,7 +335,7 @@ def trajectory_from_index(self, index):
         """
         mask = np.isin(self.indices, (index,), assume_unique=True)
         if not np.any(mask):
-            print("The particle index %d is not in the list!" % (index))
+            print(f"The particle index {index} is not in the list!")
```

Comment:
i know you're going for a 1:1 refactor here... but what about moving this string to the actual `IndexError` below?

Reply:
Good idea, but I'd prefer to do it in a follow up PR. I'm worried about making a breaking change for anyone and unintentionally hiding it behind a giant refactor.